ethereum/scripts/register_all_chains.js

```diff
@@ -6,16 +6,19 @@ const NFTBridge = artifacts.require("NFTBridgeEntrypoint");
 const BridgeImplementationFullABI = jsonfile.readFileSync(
   "../build/contracts/BridgeImplementation.json"
 ).abi;
-const solTokenBridgeVAA = process.env.REGISTER_SOL_TOKEN_BRIDGE_VAA;
-const solNFTBridgeVAA = process.env.REGISTER_SOL_NFT_BRIDGE_VAA;
-const terraTokenBridgeVAA = process.env.REGISTER_TERRA_TOKEN_BRIDGE_VAA;
-const terraNFTBridgeVAA = process.env.REGISTER_TERRA_NFT_BRIDGE_VAA;
-const terra2TokenBridgeVAA = process.env.REGISTER_TERRA2_TOKEN_BRIDGE_VAA;
-const bscTokenBridgeVAA = process.env.REGISTER_BSC_TOKEN_BRIDGE_VAA;
-const algoTokenBridgeVAA = process.env.REGISTER_ALGO_TOKEN_BRIDGE_VAA;
-const nearTokenBridgeVAA = process.env.REGISTER_NEAR_TOKEN_BRIDGE_VAA;
-const wormchainTokenBridgeVAA = process.env.REGISTER_WORMCHAIN_TOKEN_BRIDGE_VAA;
-const aptosTokenBridgeVAA = process.env.REGISTER_APTOS_TOKEN_BRIDGE_VAA;
+
+// The input parameter is a RegExp
+// It returns an array of process.env variables satisfying the input RegExp
+function getFilteredEnvs(regexp) {
+  const filteredEnvs = [];
+  for (const [key, value] of Object.entries(process.env)) {
+    if (regexp.test(key) && value) {
+      console.log("getFilteredEnvs: pushing " + key);
```

Comment:
I feel like we don't need this log here

Reply:
Could go either way, was kinda nice to see which bridges were getting registered (though maybe change the log output). I’m 🤷‍♂️. Need this for tests though so I’ll merge and we can revisit in the next “make CI have less output” event. 🙃

Reply:
agreed it's nice to see which chain is getting registered, but the log shouldn't be emitted by the helper function that grabs the env var